### PR TITLE
Feat post page

### PR DESCRIPTION
**Pull request by anangyoga**
**Changes/Features:**
I created the Post page based on the UI Design. I also created a small routing where we can see the Post page.

**Testing:**
1. Upvotes Component
- I tested for component renders
- I tested for both buttons render
- I tested to display correct props count

2. Homepage
- I tested for Homepage renders
- I tested for Homepage to have 3 columns

3. Post
- I tested for Post renders
- I tested for Post has hero image
- I tested for Post has 6 images
- I tested for users can write comment

### CODE REVIEW

**Comment by anangyoga** [comment]
hi kak @fatjan this PR is ready for Review.
I've made the test and all passed.

unit-test

File: src/data/chipLists.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const chipLists = [
```
Comment (fatjan) [inline src/data/chipLists.ts]: Is there a specific reason of making this file a ts one not js?

also the other file in the same dir

File: src/pages/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Container, Grid } from "@mui/material";
```
Comment (fatjan) [inline src/pages/Homepage.jsx:2]: please also add a test file for this file

File: src/pages/Post.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
+import { ImageList, ImageListItem, Box, Card, CardMedia, Container, Grid, Typography, Chip, IconButton, Paper, Stack, TextField, Button, Avatar } from "@mui/material";
```
Comment (fatjan) [inline src/pages/Post.jsx]: please also add a test file for this file

**Comment by fatjan** [comment]
> _ No description provided. _

Please add some description in your next PRs, thanks

File: src/pages/Homepage.jsx (hunk 2 above)
Comment (anangyoga) [reply src/pages/Homepage.jsx:2]: I have tested this file.

File: src/pages/Post.jsx (hunk 3 above)
Comment (anangyoga) [reply src/pages/Post.jsx]: I also have tested this.

**Review by anangyoga — COMMENTED** [review]
hi kak, I have made some changes based on your suggestion. let me know if there's more I can do. thank you.

**Comment by fatjan** [comment]
> **Changes/Features:** I created the Post page based on the UI Design. I also created a small routing where we can see the Post page.
> 
> **Testing:**
> 
> 1. Upvotes Component
> 
> * I tested for component renders
> * I tested for both buttons render
> * I tested to display correct props count
> 
> 2. Homepage
> 
> * I tested for Homepage renders
> * I tested for Homepage to have 3 columns
> 
> 3. Post
> 
> * I tested for Post renders
> * I tested for Post has hero image
> * I tested for Post has 6 images
> * I tested for users can write comment

Great description, would be better to include the ss of what you did as well. Let's do it on the next PR.

Example:
Screenshot 2024-01-25 at 16 51 04

Is this correct or am I missing something here?

**Comment by fatjan** [comment]
> > **Changes/Features:** I created the Post page based on the UI Design. I also created a small routing where we can see the Post page.
> > **Testing:**
> > 
> > 1. Upvotes Component
> > 
> > 
> > * I tested for component renders
> > * I tested for both buttons render
> > * I tested to display correct props count
> > 
> > 
> > 2. Homepage
> > 
> > 
> > * I tested for Homepage renders
> > * I tested for Homepage to have 3 columns
> > 
> > 
> > 3. Post
> > 
> > 
> > * I tested for Post renders
> > * I tested for Post has hero image
> > * I tested for Post has 6 images
> > * I tested for users can write comment
> 
> Great description, would be better to include the ss of what you did as well. Let's do it on the next PR.
> 
> Example: Screenshot 2024-01-25 at 16 51 04
> 
> Is this correct or am I missing something here?

This is what I see when running the app using your branch

**Comment by fatjan** [comment]
> > > **Changes/Features:** I created the Post page based on the UI Design. I also created a small routing where we can see the Post page.
> > > **Testing:**
> > > 
> > > 1. Upvotes Component
> > > 
> > > 
> > > * I tested for component renders
> > > * I tested for both buttons render
> > > * I tested to display correct props count
> > > 
> > > 
> > > 2. Homepage
> > > 
> > > 
> > > * I tested for Homepage renders
> > > * I tested for Homepage to have 3 columns
> > > 
> > > 
> > > 3. Post
> > > 
> > > 
> > > * I tested for Post renders
> > > * I tested for Post has hero image
> > > * I tested for Post has 6 images
> > > * I tested for users can write comment
> > 
> > 
> > Great description, would be better to include the ss of what you did as well. Let's do it on the next PR.
> > Example: Screenshot 2024-01-25 at 16 51 04
> > Is this correct or am I missing something here?
> 
> This is what I see when running the app using your branch

Please rename the page to Review instead of Post

And here's what it looks like:
I think it would be better if we add some spaces on the left and right, we can discuss more on this during tonight's meeting
Screenshot 2024-01-25 at 17 39 37

The figma:
Screenshot 2024-01-25 at 17 38 12

**Comment by anangyoga** [comment]
hi kak @fatjan there are some changes I did:
- I have changed the /post path to /review.

- I have added temporary 'button' on the homepage to navigate to /review page.
review


- I also have adjusted the left-right padding to match the Footer's padding. I can increase or decrease it now but so far, I make it has same padding with Footer component.
padding

I'm open for any revision. Thank you.

**Review by fatjan — APPROVED** [review]

**Pull request merged**